src/index.js

```diff
@@ -24,19 +24,17 @@ module.exports = {
     'block-spacing': 'off',
     'brace-style': 'off',
     camelcase: 'off',
-    'comma-dangle': 'off',
```

Comment:
Because they are already set on the `recommended` config that is being inherited.